legend-engine-xt-nonrelationalStore-mongodb-pure/src/main/resources/core_nonrelational_mongodb/metamodel/mongodbRuntime.pure

```diff
@@ -53,4 +54,18 @@ Class meta::external::store::mongodb::metamodel::runtime::MongoDBURL
 {
     baseUrl: String[1];
     port: Integer[1];
-}
\ No newline at end of file
+}
+
+Enum meta::external::store::mongodb::metamodel::authentication::AuthenticationMechanism
```

Comment:
How is this use?  Also, can we delete the commented code or we plan to bring that concept at some point?

Reply:
These authentication enums are what Mongo offers as external authentication support mechanism.  So they map 1-1 with the Mongo values.  But given that we support only username-passwd & kerberos (via GSSAPI only) the enums are not set